updating/understanding_updates/intro-to-updates.adoc

```diff
@@ -28,6 +28,13 @@ As the CVO applies a manifest to a cluster Operator, the Operator might perform
 The CVO monitors the state of each applied resource and the states reported by all cluster Operators. The CVO only proceeds with the update when all manifests and cluster Operators in the active Runlevel reach a stable condition.
 After the CVO updates the entire control plane through this process, the Machine Config Operator (MCO) updates the operating system and configuration of every node in the cluster.
 
+include::modules/update-availability-faq.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels-releases[Understanding update channels and releases]
+
```

Comment:
@shellyyang1989 this is the nearest place where I could add the link for the "How cluster updates work" section, would that work for you?

Reply:
Ah, looking at the [section](https://docs.openshift.com/container-platform/4.13/updating/understanding_updates/intro-to-updates.html), we already have this reference in the end. So no need to add it once again and please ignore my comments^.

Additional resources
For more detailed information about each major aspect of the update process, see [How cluster updates work](https://docs.openshift.com/container-platform/4.13/updating/understanding_updates/how-updates-work.html#how-updates-work).

By the way, we have 2 additional resources in the section. Not sure if we intend to do it. 

Additional resources
[Machine config overview](https://docs.openshift.com/container-platform/4.13/post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks)

[Using the OpenShift Update Service in a disconnected environment](https://docs.openshift.com/container-platform/4.13/updating/updating-restricted-network-cluster/restricted-network-update-osus.html#update-service-overview_updating-restricted-network-cluster-osus)

[Update channels](https://docs.openshift.com/container-platform/4.13/updating/understanding-upgrade-channels-release.html#understanding-upgrade-channels_understanding-upgrade-channels-releases)

Additional resources
For more detailed information about each major aspect of the update process, see [How cluster updates work](https://docs.openshift.com/container-platform/4.13/updating/understanding_updates/how-updates-work.html#how-updates-work).

Reply:
Yes I believe that's intentional. "Additional Resource" sections with larger headings (that also show up on the page TOC on the right) are additional resources for the entire page, while sections with smaller headings are meant to be additional resources for the specific section they follow.